keyboards/handwired/replicazeron/stm32f103/keyboard.json

```diff
@@ -0,0 +1,32 @@
+{
+    "development_board": "bluepill",
+    "features": {
+        "bootmagic": true,
+        "console": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true
+    },
+    "matrix_pins": {
+        "cols": ["A7", "A6", "A5", "A4", "B4"],
+        "rows": ["B15", "A8", "A9", "A10", "A15", "B3"]
+    },
+    "rgblight": {
+        "led_count": 6
+        "animations": {
+            "breathing": true,
+            "knight": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "rgb_test": true,
```

Comment:
Probably don't need the test animation. 
```suggestion
```

Reply:
You are right. I put that in there when I was testing the hardware and never removed it.

Reply:
@drashna Thanks again for your help :)